include/seqan3/alignment/pairwise/alignment_result.hpp

```diff
@@ -114,6 +124,8 @@ namespace seqan3
  * \if DEV
  * To access the type of the passed alignment result value use the seqan3::detail::alignment_result_value_type_accessor
  * transformation trait.
+ * In the new implementation the seqan3::detail::policy_alignment_result_builder is used to set the respective
```

Comment:
How will this comment age? How long will this be "new"?

Reply:
Until the old implementation can be removed completely.